relay-server/src/actors/processor.rs

```diff
@@ -399,7 +408,11 @@ fn track_sampling_metrics(
             sdk_version = sdk_version,
         );
 
-        let percentage = ((change as f64) / (total as f64)).min(1.0);
+        let percentage = match (change, total) {
+            (0, 0) => 0.0, // 0% indicates no premature changes.
+            _ => ((change as f64) / (total as f64)).min(1.0) * 100.0,
```

Comment:
nit: Doing the x100 here instead of on the presentation layer seems wrong.

Reply:
Agreed, though I have not found a way to do this in Datadog. If you prefer, we can also keep it between 0-1

Reply:
I think it's fine - the name says "percentage", so multiplying by 100 does make the number consistent with the metric name.